src/main/resources/tools.json

```diff
@@ -9,17 +9,17 @@
       "silentMode": true,
       "platforms": {
         "win": {
-          "url": "https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.1.0/odo-windows-amd64.exe.zip",
+          "url": "https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.2.0/odo-windows-amd64.exe.zip",
```

Comment:
I guess the versions in `tools.odo.version` and `tools.odo.versionMatchRegExpr` need to be updated to `3.2.0` too, no?